ocs_ci/utility/azure_utils.py

```diff
@@ -315,28 +317,48 @@ def get_disk_state(self, volume_name):
             self.cluster_resource_group, volume_name
         ).disk_state
 
-    def start_vm_instance(self, vm_name):
+    def start_vm_instance(self, node_names, timeout=540, wait=True):
         """
         Start an Azure vm instance
 
         Args:
-            vm_name: Name of azure vm instance
+            node_names(list): List of azure vm instance to start
+            wait (bool): True for waiting the instances to start, False otherwise
+            timeout (int): time in seconds to wait for node to reach 'ready' state.
 
         """
-        result = self.compute_client.virtual_machines.start(
-            self.cluster_resource_group, vm_name
-        )
-        result.wait()
 
-    def stop_vm_instance(self, vm_name):
+        for node_name in node_names:
+            result = self.compute_client.virtual_machines.start(
+                self.cluster_resource_group, node_name
+            )
+            result.wait()
+
+        if wait:
+            # When the node is reachable then the node reaches status Ready.
+            logger.info(f"Waiting for nodes: {node_names} to reach ready state")
+            wait_for_nodes_status(
```

Comment:
Node related operations should not be done in this file. This file should be for Azure resources related operations and nodes is an Openshift resource. Please follow the same convention we have in other platforms' low level modules:
https://github.com/red-hat-storage/ocs-ci/blob/a6902fa00f24d5b5fc4bc0939a1f2249e537e57d/ocs_ci/utility/aws.py#L471
https://github.com/red-hat-storage/ocs-ci/blob/a6902fa00f24d5b5fc4bc0939a1f2249e537e57d/ocs_ci/utility/vsphere.py#L442

Reply:
Yes, this code wrap is not appropriate. I am closing this PR